Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,7 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+    <Reference Include="MGCBReferences" />
```

Comment:
@DDReaper - You can remove this and the `MGCBReferences.definition` all together.  Just needs to reference `MonoGame.Framework.Core` here.